### DISCUSSION TITLE
List of Open Issues on Topic of Information Content Entities

### DISCUSSION THREAD

**Discussion by rorudn** [CommonCoreOntology/CommonCoreOntologies#324] [Announcements]
For convenience of discussions to follow, here is a list of open issues in descending numerical order on the topic of ICEs:

#184 Spatial and temporal reference systems are directive – Move spatial and temporal reference systems from subclasses of descriptive information content entity to prescriptive as these systems prescribe how to create a geospatial and temporal points.

#160 Deconflicting IBEs and ICEs – this seems to be a simple change of definitions. Change definition of List from “An Information Bearing Artifact that represents an ordered sequence of values, where the same value may occur more than once.” To “An information bearing artifact that is a carrier of some information content entity that represents an ordered sequence of values, where the same value may occur more than once.”

#149 Does CCO contradict an example-of-usage annotation for object property RO_0000059 (concretizes)? – in the object property annotation a plan is a specifically dependent continuant where in CCO a plan is a generically dependent continuant. Calls for either resolving the conflict or explaining why one doesn’t exist.

#140 Difference between information content entity (IBE) and information content artifact (IBA)? – The intended distinction that IBEs are instances of primitive types and IBAs are compounds is not expressed in the definitions and examples are produced intending to show non-disjoint classes.

#132 Characterizing ICEs, propositions, and images – The use of propositions in the definitions of ICEs is both unnecessary and inconsistent with how ICEs are explained in https://ceur-ws.org/Vol-1515/regular10.pdf 

#124 Document context for terms that designate, and other questions – The definition of the designates property should show consideration of the context in which a designator is used and issues regarding some of the subclasses of Designative Information Content Entity

#118 Competency questions for temporal information – Calls for set of competency questions from actual uses of temporal information to help assess how such information is represented in the ontologies. Swartik provided one such set.

#114 Reference Systems and Scales – Asks how to place rating scales within the ICE hierarchy, considers two options cco:ReferenceSystem and cco:PriorityScale, points out some issues with both and how those might be mitigated.

**Comment by neilotte** [comment]
This is a very helpful list. As these more strategic discussion items are being moved to the discussion forum, I am converting this to a discussion now where these items may continue to be tracked.
